apps/src/applab/designMode.js

```diff
@@ -250,6 +250,10 @@ designMode.updateProperty = function (element, name, value) {
       break;
     case 'icon-color':
       element.setAttribute('data-icon-color', value);
+      const imageUrl = element.getAttribute('data-canonical-image-url');
+      if (ICON_PREFIX_REGEX.test(imageUrl)) {
+        element.src = assetPrefix.renderIconToString(imageUrl, element);
```

Comment:
Okay, it looks like there are three design element types that can use icons/images and you've currently only added support for one of them.  Let's add support for all three. (unless, @poorvasingal, that was already considered outside the scope of this change?)

Image works as advertised.  The other two elements are Buttons and Screens, and they both appear to use `backgroundImage` instead of `src` for their icons ([here](https://github.com/code-dot-org/code-dot-org/blob/2cc5c5e/apps/src/applab/designMode.js#L254-L284) and [here](https://github.com/code-dot-org/code-dot-org/blob/2cc5c5e/apps/src/applab/designMode.js#L285-L304)) because they aren't actually `img` tags under the hood.